analyzers/tests/SonarAnalyzer.UnitTest/TestCases/TestMethodShouldContainAssertion.NUnit.cs

```diff
@@ -519,6 +521,53 @@ public void ReceivedInOrder()
         }
     }
 
+    // All assert methods start with "Should*". We do not test all APIs, just some.
+    public class ShouldlyTests
+    {
+        int i = 1;
+        public dynamic Foo() => null;
+        public void Bar() { }
+
+        [TestCase]
```

Comment:
We don`t need this NoAssert case, as it is already tested and has nothing specific to Shoudly.